src/vse_sync_pp/parsers/phc2sys.py

```diff
@@ -7,31 +7,35 @@
 
 from .parser import (Parser, parse_decimal)
 
+
 class TimeErrorParser(Parser):
     """Parse time error from a phc2sys log message"""
     id_ = 'phc2sys/time-error'
     elems = ('timestamp', 'terror', 'state', 'delay')
     y_name = 'terror'
     parsed = namedtuple('Parsed', elems)
+
     @staticmethod
     def build_regexp():
         """Return a regular expression string for parsing phc2sys log file lines"""
         return r'\s'.join((
-            r'^phc2sys' +
-            r'\[([1-9][0-9]*\.[0-9]{3})\]:'+  # timestamp
+            r'^phc2sys' + # noqa: W504
```

Comment:
Instead of adding a noqa I'd move `+`

Reply:
Fixed, added the opposite `W503` to the ignored list